src/components/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+
+import { cn } from '@/lib/utils'
+import * as CollapsiblePrimitive from '@radix-ui/react-collapsible'
+import React from 'react'
+import { SlArrowDown, SlArrowUp } from 'react-icons/sl'
+
+const Collapsible = CollapsiblePrimitive.Root
+const CollapsibleTrigger = React.forwardRef<
+  React.ElementRef<typeof CollapsiblePrimitive.Trigger>,
+  React.ComponentPropsWithoutRef<typeof CollapsiblePrimitive.Trigger>
+>(({ className, children, ...props }, ref) => {
+  const iconBaseCn = 'h-6 w-6 shrink-0'
+  return (
+    <div className="flex">
+      <div className="flex items-center justify-center gap-[10px]">
+        <CollapsiblePrimitive.Trigger
+          ref={ref}
+          className={cn('transition-all hover:underline group', className)}
```

Comment:
not a big deal, just thought I'd mention that the underline on hover does not seem to work

Reply:
Indeed it's unnecessary without text 👍🏼